backend/apps/authentication/views.py

```diff
@@ -117,3 +124,68 @@ def get(self, request, uid, token):
             return Response({
                 'detail': 'Invalid verification link.'
             }, status=status.HTTP_400_BAD_REQUEST)
+
+
+class PasswordResetRequestView(APIView):
+    """
+    Endpoint to request a password reset.
+    POST /api/auth/password-reset/
+    Request body:
+    {
+        "email": "user@example.com"
+    }
+    Sends an email with a link to reset the password if a user with
+    the provided email exists.
+    Always returns 200 with a generic message to prevent account enumeration.
+    """
+    permission_classes = [AllowAny]
+    serializer_class = PasswordResetRequestSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+
+        if serializer.is_valid():
+            user = User.objects.get(
+                email=serializer.validated_data['email']
+            )
+
+            token = default_token_generator.make_token(user)
+            uid = urlsafe_base64_encode(force_bytes(user.id))
+            reset_link = f'{settings.FRONTEND_URL}/reset-password?uid={uid}&token={token}'
+
+            send_password_reset_email(user, reset_link)
+
+        # always return success to avoid account enumeration
+        return Response(
+            {'detail': 'Password reset link sent.'},
+            status=status.HTTP_200_OK
+        )
```

Comment:
The view builds reset_link using settings.FRONTEND_URL without validation; if FRONTEND_URL is missing, the view will raise AttributeError. Also DEFAULT_FROM_EMAIL may be missing causing email send to fail.